config/src/main/resources/application.base.conf

```diff
@@ -31,6 +31,15 @@ akka {
       connecting-timeout = 2s
     }
   }
+  kafka {
+    producer {
```

Comment:
Regarding maximum batch size, the size of a msgPack is `48,144` bytes.
The default maximum batch size is `1048576`, so probably no problem.